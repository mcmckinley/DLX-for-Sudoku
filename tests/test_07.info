Extreme 9x9 