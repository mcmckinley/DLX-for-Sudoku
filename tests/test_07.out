9  3  4   1  7  2   6  5  8  
5  6  1   9  4  8   3  7  2  
7  2  8   6  3  5   9  1  4  

4  1  7   2  6  9   5  8  3  
8  5  3   4  1  7   2  9  6  
2  9  6   5  8  3   7  4  1  

1  4  9   7  2  6   8  3  5  
3  7  2   8  5  1   4  6  9  
6  8  5   3  9  4   1  2  7  
